python/ray/rllib/agents/ppo/appo.py

```diff
@@ -5,13 +5,15 @@
 from ray.rllib.agents.ppo.appo_policy import AsyncPPOTFPolicy
 from ray.rllib.agents.trainer import with_base_config
 from ray.rllib.agents import impala
+from ray.rllib.optimizers import AsyncSamplesOptimizer
+from ray.rllib.evaluation.rollout_worker import RolloutWorker
 
 # yapf: disable
 # __sphinx_doc_begin__
 DEFAULT_CONFIG = with_base_config(impala.DEFAULT_CONFIG, {
     # Whether to use V-trace weighted advantages. If false, PPO GAE advantages
     # will be used instead.
-    "vtrace": False,
+    "vtrace": True,

```

Comment:
why default to True?

Reply:
APPO works better with vtrace set to true; default Surrogate loss is biased